### PR TITLE
Home and about page

### PR DESCRIPTION
**Pull request by diegoyon**
### Home and about page

- Created the home and about page using the design guidelines by Cindy Shin in Behance (Creative Commons licensed)
- Topic is personalized to be about the Ultra Music Festival 2023.
- Added responsive design for mobile and desktop using CSS media queries.
- Added links to the home page and about page to toggle between these two pages.
- Added mobile menu using Javascript.
- Dynamically created the featured DJs section using Javascript.
- [Link to video](https://www.loom.com/share/a082624c329c4d3398dc9016b99a03c5)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          <li><a href="./index.html#program">Program</a></li>
+          <li><a href="#Join">Join</a></li>
+          <li><a href="#Sponsor">Sponsor</a></li>
+          <li><a href="#News">News</a></li>
+        </ul>
+        <div class="campaign">
+          <a href="#CCCampaign">CC Campaign</a>
+        </div>
+      </div>
+  </header>
```
Comment (gemmen29) [inline about.html]: - [ ] Kindly add the black nav that exists in the Behance design in the home and about pages.
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    </div>
+    <button>Buy tickets UMF 2022</button>
+    <a class= "whole-program" href="#">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section class="djs">
+    <h3>Featured DJ's</h3>
+    <hr class="title-rule">
+    <div class="grid-container">
+      <!-- Dynamically added items -->
+    </div>
```
Comment (gemmen29) [inline index.html:92]: - [ ] Kindly add the background image(black and white) in the speaker section to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

File: about.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      <img src="./logos/guetta.png" alt="guetta">
+      <img src="./logos/hardwell.png" alt="hardwell">
+      <img src="./logos/snake.png" alt="snake">
+      <img src="./logos/tiesto.png" alt="tiesto">
+    </div>
+  </section>
+  <footer>
+    <img src="./logos/ultra.png" alt="ultralogo">
+    <p>Ultra Music Festival. All Rights Reserved</p>
+  </footer>
```
Comment (gemmen29) [inline about.html]: - [ ] Please make the partner section disappear and change the background to dark on the `about` page to match the Behance design.

| Your design | Behance design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @diegoyon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @diegoyon 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**